test/testcafe/spec/ChallengeAuthenticatorOktaVerifyOnly_spec.js

```diff
@@ -0,0 +1,51 @@
+import {RequestMock} from 'testcafe';
+import SelectAuthenticatorPageObject from '../framework/page-objects/SelectAuthenticatorPageObject';
+import xhrOktaVerifyOnlyMethodsWithDeviceKnown
+  from '../../../playground/mocks/data/idp/idx/authenticator-verification-data-ov-only-with-device-known';
+import xhrOktaVerifyOnlyMethodsWithoutDeviceKnown
+  from '../../../playground/mocks/data/idp/idx/authenticator-verification-data-ov-only-without-device-known';
+
+const FORM_TITLE = 'Verify it\'s you with an authenticator';
+const FORM_SUBTITLE = 'Select from the following options';
+const ENTER_CODE_TEXT = 'Enter a code';
+const FAST_PASS_TEXT = 'Use Okta Verify on this device';
+const PUSH_NOTIFICATION_TEXT = 'Get a push notification';
+
+const mockChallengeOVOnlyMethodsWithDeviceKnown = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrOktaVerifyOnlyMethodsWithDeviceKnown);
+
+const mockChallengeOVOnlyMethodsWithoutDeviceKnown = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrOktaVerifyOnlyMethodsWithoutDeviceKnown);
+
+fixture('Challenge Authenticator Okta Verify Only');
```

Comment:
nit: We can merge these tests with `ChallengeAuthenticatorOktaVerify_spec`. That also tests when OV is the only authenticator. 

Probably all mocks - `authenticator-verification-okta-verify-select-method` mock and `authenticator-verification-data-ov-only-without-device-known` and `authenticator-verification-data-ov-only-with-device-known` mocks can be merged.

The only differences I see are:
- method ordering, which can be changed.
- and `device known`, which can be set in the test itself instead of creating a separate mock file.

Reply:
+1

Reply:
I merged test + mock files together